grakn-graql/src/test/java/ai/grakn/graql/internal/reasoner/inference/GenealogyTest.java

```diff
@@ -293,7 +294,7 @@ public void testWife2(){
 
     @Test
     public void testSiblings() {
```

Comment:
Crappy test names in this class

Reply:
strictly use-case-based :p

Reply:
actually it's sort of an integration test